src/dotnet/SnippetGenerator/SnippetGenerator.csproj

```diff
@@ -0,0 +1,23 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <Description>Generates code snippets for readmes in the azure-sdk-for-net repo.</Description>
+    <TargetFramework>net5.0</TargetFramework>
```

Comment:
https://github.com/Azure/azure-sdk-tools/blob/main/tools/pipeline-generator/Azure.Sdk.Tools.PipelineGenerator/Azure.Sdk.Tools.PipelineGenerator.csproj#L7 Why not package it as tool?